bigtable/metricscaler/metricscaler_test.py

```diff
@@ -35,6 +36,8 @@
 INSTANCE_ID_RANGE = 10000
 BIGTABLE_INSTANCE = INSTANCE_ID_FORMAT.format(
     random.randrange(INSTANCE_ID_RANGE))
+BIGTABLE_DEV_INSTANCE = INSTANCE_ID_FORMAT.format(
+    random.randrange(INSTANCE_ID_RANGE))
```

Comment:
We are running multiple builds simultaneously.  What happens when two builds are using the same value? Can you do `INSTANCE_ID_FORMAT.format(uuid.uuid4())`?

Reply:
good idea

Reply:
Oops sorry, we're getting `Invalid id for collection instances : Length should be between [6,33], but found 54 'metric-scale-test-2dbdbbc5-5a3f-49be-86f4-4645bcc099f2'"`

Maybe use the first part of the uuid, or something to make it shorter?

Reply:
@billyjacobson I think `uuid.uuid4().hex` works.